tests/pages/vote-page.js

```diff
@@ -7,13 +7,15 @@ export default createPage({
     hoverOnDevelopmentStatusLabel: triggerable('mouseenter', '[data-test-development-status-label]'),
     name: text('[data-test-course-extension-idea-name]'),
     voteButtonText: text('[data-test-vote-button]'),
+    isGreyedOut: (card) => card.hasClass('opacity-50'),
```

Comment:
When there's no clear order or order doesn't matter, we sort things alphabetically - so this would appear after `hoverOn...`. This helps avoid that extra decision around where to place a line, also helps keep things more consistent across the codebase

Reply:
Addressed in later commits!